identity_storage/src/secret_manager_wrapper.rs

```diff
@@ -0,0 +1,32 @@
+// Copyright 2020-2023 IOTA Stiftung
+// SPDX-License-Identifier: Apache-2.0
+
+use iota_sdk::client::secret::stronghold::StrongholdSecretManager;
+use iota_sdk::client::secret::SecretManager;
+use iota_stronghold::Stronghold;
+use std::sync::Arc;
+use tokio::sync::MutexGuard;
+
+/// Wrapper around `SecretManager` that implements the storage interfaces.
```

Comment:
```suggestion
/// Wrapper around a `StrongholdSecretManager` that implements the [`KeyIdStorage`](todo:link) and [`JwkStorage`](todo:link) interfaces.
```

Yes, technically it wraps a `SecretManager`, but conceptually it's a `StrongholdSecretManager` and that is what matters to the user.